src/package.xml

```diff
@@ -4397,4 +4397,4 @@
         <name>WorkflowRule</name>
     </types>
     <version>48.0</version>
-</Package>
+</Package>
```

Comment:
looks like u r accidentally commiting a change to package.xml file. You might want to discard it.

Reply:
Thanks. I will discard it.